csm-1.5/CASMTRIAGE-7166-tapms-tenants-deploying/README.md

```diff
@@ -0,0 +1,38 @@
+# CASMTRIAGE-7166 - Fix for the status field in tenant deployments
+
+Fixes an issue with TAPMS where it will try to add nodes to the tenants HSM node group more than once, causing the tenant to get stuck in a "deploying" state.
+
+## Prerequisites
+
+- CSM versions 1.5.0 to 1.5.2
+
+## JIRA(s)
+
+This hotfix covers the following JIRA(s):
+
+* [CASMTRIAGE-7166](https://jira-pro.it.hpe.com:8443/browse/CASMTRIAGE-7166)
+
+
+## Known Issues
+
+- With this fix in place it is possible to create tenants that request the same nodes.
+  Both tenants in this case will show as deployed, but the nodes will only belong to the HSM group of the first tenant created.
+  This failure is silent, so it is up to the admin creating the tenant definition to ensure that the tenant definitions are valid.
+
+## Usage
+
+### Interactive
+
+Run the script without arguments to witness a deterrent, and interactively accept the hotfix.
+
+```bash
+./install-hotfix.sh
+```
+
+### Non-interactive
+
+Run the script with `-y` to proceed non-interactively.
+
+```bash
+./install-hotfix.sh -y
+```
```

Comment:
I don't see any code in your install script which actually parses the '-y' argument, unless I am missing something.

````suggestion
```bash
./install-hotfix.sh
```
````